demo-circuit/src/naive_threshold_sig_w_key_rotation/data_structures.rs

```diff
@@ -9,6 +9,9 @@ use primitives::{
 
 use crate::{common::NULL_CONST, Error};
 
+pub const VALIDATOR_HASH_SALT: u8 = 0u8;
+pub const INITIAL_EPOCH_ID: u32 = 0;
```

Comment:
I'm not fully comfortable in putting a meaningful value for something that is supposed to be dummy. It's true that we rely on the MC being honest and putting the correct dummy value, but I'm also thinking about some possible replay attacks for the first certificates.
But maybe, again, I'm overthinking it